kge/job/search.py

```diff
@@ -130,8 +131,12 @@ def _run_train_job(sicnk, device=None):
                 train_job_config.get("job.device"),
             )
         )
-        job = Job.create(train_job_config, search_job.dataset, parent_job=search_job)
-        job.resume()
+        job = Job.resume(
```

Comment:
Confusing that this is called resume (and not create). It does not actually resume anything, but loads a job. So why not load_from?

Reply:
now called find_and_create. If you have a better idea for the name, let me know